src/llmcompressor/arg_parser/README.md

```diff
@@ -0,0 +1,45 @@
+# Argument parser for `oneshot`, `train`, `eval` entrypoints
+
+Argument parsers in `llm-compressor` define the input arguments required for various entrypoints, including `oneshot`, `train`, and `eval`.
```

Comment:
arg_parser makes it sound like it's a cli tool.
I would just call this args

Reply:
Isn't this an arg parser?
https://github.com/huggingface/transformers/blob/main/src/transformers/hf_argparser.py

Reply:
They are actually the parsing the args in that file. 
The files in this folder are just dataclasses. 

We should update the name